src/app.tsx

```diff
@@ -60,6 +60,17 @@ export const App = () => {
     [theme],
   );
 
+  useEffect(() => {
```

Comment:
What's the reason of extracting this into dedicated `useEffect`?

Reply:
`app.onboarded` does not work as expected, and sometimes, while on the welcome screen, setting `app.onboarded` to `true` does not trigger a transition to the home screen